src/Components/NewTask/Category.js

```diff
@@ -1,77 +1,49 @@
-import React, { Component } from "react";
-import Downshift from "downshift";
+import React from "react";
+import {withRouter} from 'react-router-dom'
+import { connect } from "react-redux"
+import { Dropdown, DropdownButton } from 'react-bootstrap';
 
-
-class Category extends Component {
+class Category extends React.Component {
```

Comment:
Does this work better when we don't import Component directly?

Reply:
It is the same. I changed it from a class to a function and then back to a class. Didnt realize it was slightly different.